README.md

```diff
@@ -213,6 +213,33 @@ export default function serverRender(req, res) => {
   `)
 ```
 
+> NOTE: this requires that the bundling and rendering happen within the same context. The module, react-universal-component/server holds a global cache of all the universal components that are rendered and makes them available via `flushChunkNames`
```

Comment:
yay!  figured it out.  i had atom-beautify and prettier-atom both installed grrr